test/UnitBadDocLoad.cpp

```diff
@@ -82,6 +82,7 @@ UnitBase::TestResult UnitBadDocLoad::testBadDocLoadFail()
         auto response = helpers::getResponseString(socket, "error:", testname);
         if (Util::startsWith(response, "error: cmd=notasync"))
         {
+            TST_LOG("Ignoring initial notasync error - waiting again");
```

Comment:
Since the alert is no longer thrown on document load, can we get rid of this extra section?

Reply:
Sure why not - with the other patch included.